news/2020-04-20-monthly-beatmapping-contest-3-results.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest #3: Results"
+date: 2020-04-20 23:00:00 +0000
+---
+
+Check out the winning submissions from the third monthly beatmapping contest!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The **Monthly Beatmapping Contest** (***MBC***) has been running strong since the start of the year! Take a peek at the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest) for information on anything and everything MBC-related.
+
+## Winners
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/91). The limitation for this month's submissions was *no slider velocity changes.*
+
+Voting resulted in the closest competition we've seen yet! While there was a 20 vote difference between second and third places, there was only a **2** vote difference between first and second!
+
+Receiving a hefty **85** votes, [Sylas](https://osu.ppy.sh/users/3906405) (previously known as TheKoala) managed to top the community vote poll with his entry of [Sound Souler - Bubble Flower](https://osu.ppy.sh/beatmapsets/1152182#osu/2404833) and is **this month's winner!**
+
+Given this contest's submissions could not change slider velocity, deciding the map's base slider velocity was a huge factor for map design. Sylas chose nearly the fastest slider speed out of all submissions, so the map's first half featured mainly circles to fit the song's low-intensity intro, while the second half distinctly contrasted with high speed/intensity sliders. Words can't express this as well as playing it yourself though — try it [here](https://osu.ppy.sh/beatmapsets/1152182#osu/2404833)!
```

Comment:
don't really need to link the map twice?
(not an important fix)

Reply:
i think it's okay in this instance given the format for winning paragraphs - the double inclusion is essentially signaling extra emphasis given the notability expressed prior